src/main/java/Duke.java

```diff
@@ -6,5 +8,55 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        System.out.println("Hello! I'm Duke\n");
+        System.out.println("What can I do for you?\n");
+
+        Scanner scan = new Scanner( System.in );
```

Comment:
Can remove the space in "... new Scanner( System.in );" to "... new Scanner(System.in);"